elements/nuxeo-admin/nuxeo-distribution-analytics.js

```diff
@@ -111,25 +151,59 @@ Polymer({
       type: Number,
       value: 1,
     },
+    disableThreshold: {
+      type: Number,
+      value:
+        Nuxeo &&
+        Nuxeo.UI &&
+        Nuxeo.UI.config &&
+        Nuxeo.UI.config.analytics &&
+        Nuxeo.UI.config.analytics.disableThreshold &&
+        Number(Nuxeo.UI.config.analytics.disableThreshold),
```

Comment:
since we have more elements being contributed to `ANALYTICS_PAGES`, perhaps we should add more namespacing? Something like:
```suggestion
        Number(Nuxeo.UI.config.analytics.documentDistribution.disableThreshold),
```
WDYT?